plugins/chat/config/locales/client.en.yml

```diff
@@ -304,6 +306,8 @@ en:
         save: "Save"
         saved: "Saved"
         unfollow: "Leave"
+        admin_title: "Admin"
+        retention_info: "Chat history will be saved for 90 days."
```

Comment:
This needs an interpolated value `%{days}` since it won't always be 90, and we need to pass `this.siteSettings.chat_channel_retention_days` into the translation in the template.